hardware/raspberrypi/frequency-management.md

```diff
@@ -8,6 +8,14 @@ For Raspberry Pi 3 Model B+, the PCB technology has been changed to provide bett
 
 The Raspberry Pi 4 Model B continues with the same PCB technology as the Raspberry Pi 3B+ to help dissipate excess heat. There is currently no soft limit defined.
 
+### DVFS on the Raspberry Pi 4
+
+On the Raspberry Pi 4, firmware from late November 2019 onwards implements Dynamic Voltage and Frequency Scaling. This technique (outlined on Wikipedia [here](https://en.wikipedia.org/wiki/Dynamic_voltage_scaling)) allows the Pi4 to run at lower temperatures whilst still providing the same performance.
```

Comment:
Should "Raspberry Pi 4" and "Pi4" be changed to "Raspberry Pi 4 Model B" to match the nomenclature used elsewhere on this page?

Reply:
Raspberry Pi 4B would be better, since that's the actual name of the product. (The model names were of course named after the BBC Micro, but if "model" was used as part of the product name for Pis in the past, it hasn't been for a long time).

Reply:
@andrum99 https://www.raspberrypi.org/products/raspberry-pi-4-model-b/ :wink: